### PR TITLE
Add blog post for disaggregated coordinator

### PR DESCRIPTION
**Pull request by swapsmagic**

### CODE REVIEW

**Review by highker — COMMENTED** [review]
Haven't looked into details but do have some high-level comments on the figures:
- The fonts seem to be inconsistent. Usually, it would be good to have figure fonts to be 1 size smaller than the text font size. (Or at least easy to read).
- Some of the fonts are too small from a browser.
- The font size in the boxes is a bit disproportional from the box sizes. The text is too small and boxes are too big. (Check our Presto paper https://research.facebook.com/publications/presto-sql-on-everything/ to see the font size vs box size).

Regarding the content itself, Tim and Ariel could be better reviewers than me given they are closer to the work + native English speakers lol

File: website/blog/2022-02-17-disggregated-coordinator.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+## Resource Management
+![ResourceManagement](/img/blog/2022-02-17-disaggregated-coordinator/resource-management.png)
+
+The resource manager runs in multi-master mode. To support that, coordinators post query updates to all resource managers. The resource manager aggregates this information. The coordinator polls a resource manager to fetch up to date information about resource group usage in the cluster.
+
+## Resource Group Consistency Model
+
+Resource groups in a disaggregated coordinator setup are eventually consistent.  While this may lead to over-admission in certain scenarios, in practice this is mitigated by gating the resource group to only allow queries to run when certain freshness guarantees have been met (as opposed to the previous logic of checking every millisecond).  This may mean if the cluster’s resource managers are down, then queries may be queued in the coordinator’s resource groups.  This is to ensure coordinators don’t over-admit queries in the face of resource manager unavailability.
+
+The following flags can help one tune the cluster’s resource groups to the desired consistency:
```
Comment (aweisberg) [inline website/blog/2022-02-17-disggregated-coordinator.md:60]: The flags should go into the Presto docs and be linked to from here :-) 
There shouldn't be anything to elaborate on that isn't already in the docs. Really if the blog post never existed the docs should be sufficient to explain what people need to know to enable and operate this feature (including why they would).

File: website/blog/2022-02-17-disggregated-coordinator.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+- **resource-group-runtimeinfo-refresh-interval (default: 100 ms)**
+
+  This configuration helps tune coordinator periodic polling intervals of cluster level resource group upsage from the resource manager.
+  
+## Discovery Service
+![discoveryservice](/img/blog/2022-02-17-disaggregated-coordinator/discovery-service.png)
+
+An embedded version of the discovery server runs on resource managers in a distributed mode. Discovery servers stay in sync by passing updates they receive to other discovery servers in the cluster.
+
+## Configuration
```
Comment (aweisberg) [inline website/blog/2022-02-17-disggregated-coordinator.md:77]: This should also be in the docs. Basically I am just pushing back on putting documentation in blog posts. It leaves the docs incomplete and goes out of date over time.

**Review by aweisberg — CHANGES_REQUESTED** [review]
Content looks quite good. Only have minor tweaks. And some push back on including documentation in the blog. The blog is for advertising :-)

**Comment by rohanpednekar** [comment]
@swapsmagic Thanks for your contribution, this is really great content. Do you think we can target the end of this month to publish this article?

**Comment by swapsmagic** [comment]
> @swapsmagic Thanks for your contribution, this is really great content. Do you think we can target the end of this month to publish this article?

Waiting for this PR https://github.com/prestodb/presto/pull/17597 to get merged before i can update this and refer to the official doc in the blog. Once that's done it's good to merge.

File: website/blog/2022-02-17-disggregated-coordinator.md (hunk 2 above)
Comment (aweisberg) [reply website/blog/2022-02-17-disggregated-coordinator.md:79]: Can we only link to the docs for the configuration? This is the thing that is most likely to go out of date and be incorrect so it's better not to duplicate it.

**Review by aweisberg — APPROVED** [review]
LGTM, thank you!

**Pull request merged**